### PR TITLE
Design System feature option

### PR DESCRIPTION
**Pull request by rodrigofernandozup**
Design System feature on BFF SDUI plugin

### CODE REVIEW

**Review by LG95 — CHANGES_REQUESTED** [review]
Please leave in draft or remove `main.py` until the cli v2 embedded python issue is resolved.

File: plugin.yml [hunk 1]
```diff
@@ -9,16 +9,29 @@ tags:
...
+    name: "design_system_configure"
+    default: "false"  
+  - label: "URL of the repository (SSH is recommended if it is private) with the component definitions: "
+    name: "url"
+    type: text
+    default: "git@github.com:Orangestack-com/uxservices-stackfront-design-system.git"
+    condition:
+      variable: "design_system_configure"
+      operator: "=="
+      value: true
```
Comment (LG95) [inline plugin.yml:37]:
```suggestion
      value: true

```

**Review by ze12augusto — CHANGES_REQUESTED** [review]

**Review by LG95 — APPROVED** [review]

File: main.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+class DeleteDesignSystemFiles(Template):
+    def post_hook(self, metadata: Metadata):
+        if not metadata.inputs["design_system_configure"]:
+            delete_file(metadata.target_path.joinpath('app', 'design-system.gradle.kts'))
+            delete_file(metadata.target_path.joinpath('app', 'ds-schema.json'))
+            delete_file(metadata.target_path.joinpath('app', 'run-ds-shema.bat'))
+            delete_file(metadata.target_path.joinpath('app', 'run-ds-shema.sh'))
+
+if __name__ == '__main__':
+    print("teste main")
```
[the remark this answers is not in the document]
Comment (rodrigofernandozup) [reply main.py]: tks!!

File: main.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class DeleteDesignSystemFiles(Template):
+    def post_hook(self, metadata: Metadata):
+        if not metadata.inputs["configure_design_system"]:
+            delete_file(metadata.target_path.joinpath('app', 'design-system.gradle.kts'))
+            delete_file(metadata.target_path.joinpath('app', 'ds-schema.json'))
+            delete_file(metadata.target_path.joinpath('app', 'run-ds-shema.bat'))
+            delete_file(metadata.target_path.joinpath('app', 'run-ds-shema.sh'))
+
+if __name__ == '__main__':
+
```
Comment (ze12augusto) [inline main.py:20]:
```suggestion
```

**Review by ze12augusto — CHANGES_REQUESTED** [review]

**Review by ze12augusto — APPROVED** [review]

**Review by ze12augusto — APPROVED** [review]

**Review by ricardojsilva — APPROVED** [review]

**Pull request merged**